controlpanel/api/helm.py

```diff
@@ -75,6 +75,15 @@ def upgrade_release(self, release, chart, *args):
             "upgrade", "--install", "--wait", "--force", release, chart, *args,
         )
 
+    def reset_home(self, release, chart, *args):
+        HelmRepository.update()
+        return self.__class__.execute(
+            "upgrade",
+            release,
+            chart,
+            *args,
+        )
```

Comment:
I'm not sure this method adds much to the existing `Helm.upgrade_release()` method which we use to install/upgrade an helm chart.

Any reason to not use that instead? It's doing the same thing AFAICT no?

Maybe I'm missing something.

Reply:
Originally they were slightly different. I'm not sure we need the `--wait` or `--force` flags for resetting the home. Putting aside these queries, if they're doing the same thing, I agree we should just re-use existing code.

Reply:
Yeah I think it's conceptually the same thing, the `--force` seem to help with helm idiosyncrasies and the `--wait` is mainly there because our Tools can take long to deploy but in this case it shouldn't harm. So I'd personally avoid adding another method but of course it's your call.

Reply:
Resolved in latest commit.